src/app/components/admin/all-uploads/all-uploads.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from "@angular/core";
+import { List } from "immutable";
+import { ListComponent } from "@components/harvest/pages/list/list.component";
+import { ActivatedRoute } from "@angular/router";
+import { ShallowHarvestsService } from "@baw-api/harvest/harvest.service";
+import { NgbModal } from "@ng-bootstrap/ng-bootstrap";
+import { adminCategory, adminUploadsMenuItem } from "../admin.menus";
+import { adminMenuItemActions } from "../dashboard/dashboard.component";
+
+@Component({
+  selector: "baw-all-uploads",
+  templateUrl: "../../harvest/pages/list/list.component.html",
+})
+class AllUploadsComponent extends ListComponent {
+  public constructor(
+    private modal: NgbModal,
+    private api: ShallowHarvestsService,
+    private activatedRoute: ActivatedRoute
+  ) {
+    super(modal, api, activatedRoute);
+  }
+
+  public override get project() {
+    return null;
+  }
+}
+
+// we have to use setToRoute here instead of linkToRoute because otherwise
+// the component will inherit the routes from the extended ListComponent
+AllUploadsComponent.setToRoute({
+  category: adminCategory,
+  pageRoute: adminUploadsMenuItem,
+  menus: { actions: List(adminMenuItemActions) },
+});
```

Comment:
strong code smell. 

a) What's wrong with it inheriting the routes?
b) If you convince me that's bad, then why aren't we making it so the page info list is not inherited?

Reply:
There's nothing wrong with inheriting the routes from the `ListComponent` (it doesn't break anything), but it's incorrect because the admin harvest list view is not mounted under the `/projects/:projectId/uploads` route and it doesn't have anything to do with the `/projects/:projectId/uploads` page info.

Reply:
Resolved, [routes are now inherited](https://github.com/QutEcoacoustics/workbench-client/pull/2090/files#diff-ae88cd1f8981bab0740627d13c4465ae458f290b910decac0a7db163cbbd5cd2R28-R32)